libraries/AP_Proximity/AP_Proximity_Backend.h

```diff
@@ -112,8 +107,7 @@ class AP_Proximity_Backend
     Vector2f _sector_edge_vector[PROXIMITY_SECTORS_MAX];    // vector for right-edge of each sector, used to speed up calculation of boundary
     Vector2f _boundary_point[PROXIMITY_SECTORS_MAX];        // bounding polygon around the vehicle calculated conservatively for object avoidance
 
-    // earth frame locations (i.e. detected obstacles stored as lat/lon points)
-    uint16_t _location_count;                               // number of locations held in _locations buffer
-    AP_Proximity::Proximity_Location _locations[PROXIMITY_SECTORS_MAX];   // buffer of locations
-    HAL_Semaphore_Recursive _rsem;                          // semaphore for access to _locations and _location_count
+    // database cached values to speed things up
+    float current_vehicle_bearing;
```

Comment:
Same with this, if inlined can disappear.

Reply:
fixed